api/v1beta1/machine_types.go

```diff
@@ -50,6 +50,11 @@ const (
 	// to pause reconciliation of deletion. These hooks will prevent removal of
 	// an instance from an infrastructure provider until all are removed.
 	PreTerminateDeleteHookAnnotationPrefix = "pre-terminate.delete.hook.machine.cluster.x-k8s.io"
+
+	// MachineCertificatesExpiryDateAnnotation annotation specifies the expiry date of the machine certificates.
+	// This annotation can be used my MachineHealthCheck to trigger machine rotation before certificates expire.
```

Comment:
```suggestion
	// This annotation can be used to trigger machine rotation before certificates expire via MachineHealthChecks.
```
* dropped `my`
* the previous version sounded a bit more like the MHC uses the annotation directly which is not the case (only indirectly after the date was propagated to status)

Reply:
I think we have to mention that this only works with control plane machines

Reply:
I adjusted the comment and dropped references to MachineHealthChecks.